woocommerce/woocommerce-admin
# Check active plugins before getting the PayPal onboarding status

Fixes #6593 

Check active plugins before getting the PayPal onboarding status

### Detailed test instructions:
-  Go to the WooCommerce home page
-  Open your browser console
-  Choose payment methods
-  See no error message
